homometer/templates/homometer/index.html

```diff
@@ -9,9 +9,10 @@
 		<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no" />
 		<title>Mensa-Homometer</title>
 		<link href="{% static 'style.css' %}" rel="stylesheet" />
-		<link rel="apple-touch-icon" sizes="180x180" href="{% static 'apple-touch-icon.png' %}">
-		<link rel="icon" type="image/png" sizes="32x32" href="{% static 'favicon-32x32.png' %}">
-		<link rel="icon" type="image/png" sizes="16x16" href="{% static 'favicon-16x16.png' %}">
+		<link rel="apple-touch-icon" sizes="180x180" href="{% static 'favicon-180.png' %}">
```

Comment:
```suggestion
		<link rel="apple-touch-icon" sizes="180x180" href="{% static 'favicon-180.png' %}" />
```

My bad. Some tags had `/>` and some didn't. Can you unify this and add `/` to this line and the following two? :)